### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by gamerscruffy** [BEEmod/BEE2.4#541]
Here are some suggestions I thought of.

1. Having an ingame puzzle-maker option to turn the turret into a defective one. Like a check option.

2. Have actual names for the check options for custom items. Because the names are "Auto-drop first cube" And other things. If possible, there should be a one that is easier to understand. Like the Piston Platform. Instead of having "Auto-drop first cube", we could have something like "Activate by player touch".

3. Check option to enable glass to be breakable or not. (By turrets or explosions.)

4. Being able to spawn cores and maybe connect them to rails is an option. 

5. Moving panels. (The ones with arms)

6. Panel doors. The ones that Wheatley open when you begin the escape of the tests. (This was actually an unused item that Valve was going to add.)

I will update this list and update the title to the date I updated more as I think of more suggestions.

**Comment by TeamSpen210** [comment]
Breakable glass will be an item. I can't change the checkmark names, if I could I would have done that. Cores won't be added. There are lots of moving panels and animations, be more specific. Defective turrets aren't very useful, especially considering they're also used in Old Aperture.

**Comment by gamerscruffy** [comment]
I guess I'll have to figure out which option does which. I might even make a guide on how to use each item in bee2.

**Comment by TeamSpen210** [comment]
Right-click on an item to get a description, that tells you how to use it.

**Comment by gamerscruffy** [comment]
Thanks!

**Comment by TeamSpen210** [comment]
I should write some code to automatically export them all into a manual or something.

**Comment by gamerscruffy** [comment]
May I use the icon for Bee2 as the thumbnail on my guide?

**Comment by TeamSpen210** [comment]
Sure, even if I didn't say so you're allowed to use stuff like that for that purpose...

**Comment by gamerscruffy** [comment]
http://steamcommunity.com/sharedfiles/filedetails/?id=879065525 took an hour and a half. xD

This is the first guide to installing the later versions of BEE2 where the installation process is different. I made sure to give you credit. If you would like, you could help me spread it. ^^

**Comment by TeamSpen210** [comment]
Don't place in dlc2, it's not needed. You can put them anywhere. Also it might be better to link to the releases page, so it's not tied a specific version.

**Comment by TeamSpen210** [comment]
Corridors force a specific entry/exit corridor to be used, instead of the random variants.

**Comment by gamerscruffy** [comment]
One more thing, I have recently discovered a bug that the "chamber-lock" door item is an error. Was it supposed to be the original "door" item?

**Comment by TeamSpen210** [comment]
No, it had a custom model which shows what it looks like, I need to remake that.

**Comment by gamerscruffy** [comment]
So the packages can go in any folder?

**Comment by TeamSpen210** [comment]
The same as the app location, which can be anywhere.

**Comment by gamerscruffy** [comment]
I've just updated the guide with the feedback you sent me.

**Comment by gamerscruffy** [comment]
Thanks for the help.

**Comment by TeamSpen210** [comment]
It doesn't have to be in the Portal 2 folder - it can be anywhere at all on your computer...

**Comment by gamerscruffy** [comment]
Would you like to be a contributor to the guide?

**Comment by gamerscruffy** [comment]
I'd be happy to give you permissions

**Comment by TeamSpen210** [comment]
It might be better to have docs in this repro's wiki pages.

**Comment by gamerscruffy** [comment]
I updated the list the same day we talked. 

"Panel doors. The ones that Wheatley open when you begin the escape of the tests. (This was actually an unused item that Valve was going to add.)"
